CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+
+## [2.0.0] - 2023-06-06
+WARNING: this version adds a change to the MYSQL schema! Make changes to the schema before deploying new 2.0.0 version.
+
+### Added
+ - MINOR added the ability to designate teams as "api managed" which will prevent changes to team info from being done via the UI
+### Changed
+ - MAJOR added the `api_managed_roster` column to the `team` table in the MYSQL schema. Before running 2.0.0 the MYSQL schema must be updated with the command "ALTER TABLE team ADD api_managed_roster BOOLEAN NOT NULL DEFAULT FALSE;" to prevent an exception from being thrown from the GET /api/v0/teams/{team} endpoint
```

Comment:
This could be a bit more direct. Eg it doesn't need to describe that an existing route will throw an exception.

It can just indicate the following query is needed, and it would also help readability if it was on its own line in a block-level code snippet rather than within a sentence/bullet.

Though, an even better option may to have a .sql file under `db/` for every new version which requires a schema change named after the version+date and mention the need to run this file in this section, eg say you need to `mysql < db/migration-version-DATE.sql` 

This would be more consistent and easier to maintain over time